ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/errors/KVErrors.scala

```diff
@@ -92,43 +92,6 @@ object KVErrors extends LedgerApiErrorGroup {
     "Errors that can arise due to concurrent processing of transactions in the participant."
   )
   object SubmissionRaces extends ErrorGroup() {
-
-    @Explanation(
-      "A contract with the same key has already been created by a concurrent transaction submission."
-    )
-    @Resolution(
-      "The correct resolution depends on the business flow, " +
-        "for example using the existing contract or retrying after it has been archived."
-    )
-    object ExternallyDuplicateKeys
-        extends ErrorCode(
-          id = "EXTERNALLY_DUPLICATE_KEYS",
-          ErrorCategory.InvalidGivenCurrentSystemStateOther, // It may succeed at a later time
-        ) {
-      case class Reject(
-      )(implicit loggingContext: ContextualizedErrorLogger)
-          extends KVLoggingTransactionErrorImpl(
-            cause = s"Inconsistent: ${ExternallyInconsistentTransaction.DuplicateKeys.description}"
-          )
-    }
-
-    @Explanation(
-      "An input contract key was re-assigned to a different contract by a concurrent transaction submission."
-    )
-    @Resolution("Retry the transaction submission.")
-    object ExternallyInconsistentKeys
-        extends ErrorCode(
-          id = "EXTERNALLY_INCONSISTENT_KEYS",
-          ErrorCategory.InvalidGivenCurrentSystemStateOther, // It may succeed at a later time
-        ) {
-      case class Reject(
-      )(implicit loggingContext: ContextualizedErrorLogger)
-          extends KVLoggingTransactionErrorImpl(
-            cause =
-              s"Inconsistent: ${ExternallyInconsistentTransaction.InconsistentKeys.description}"
-          )
-    }
-
```

Comment:
I think `ExternallyDuplicateKeys` and `SubmittingPartyNotKnownOnLedger` should share the fate of their brethren `ExternallyInconsistentKeys` and `ExternallyDuplicateKeys` and be moved to `CommandRejections`.

Reply:
`ExternallyDuplicateKeys` was already there no? Moved `SubmittingPartyNotKnownOnLedger` 👍